src/main.rs

```diff
@@ -143,7 +143,7 @@ async fn run(options: cli::Options) -> Result<()> {
     let pubsub = PubSubManager::init(
         options.websocket_connections,
         accounts.clone(),
-        program_accounts.clone(),
+        Arc::clone(&program_accounts),
```

Comment:
Why is this Arc out of program_accounts? It's inconsistent with accounts

Reply:
I did this, to make it explicit that cache is shared, but I guess you right, it's inconsistent with accountsdb, reverted it back